packages/flutter/test/widgets/two_dimensional_utils.dart

```diff
@@ -195,7 +195,7 @@ class RenderSimpleBuilderTableViewport extends RenderTwoDimensionalViewport {
 
   @override
   TestExtendedParentData parentDataOf(RenderBox child) {
-    return child.parentData! as TestExtendedParentData;
+    return super.parentDataOf(child) as TestExtendedParentData;
```

Comment:
This makes me wonder if we should add `@mustCallSuper` to the method in the base class

Reply:
Can you add that in this PR?